### PR TITLE
use dovecot-fts-xapian from alpine package

### PR DESCRIPTION
**Pull request by willofr**
## What type of PR?

enhancement

## What does this PR do?
use dovecot-fts-xapian from alpine packages repository (newer) instead of compiling an older version from source
see https://pkgs.alpinelinux.org/package/edge/community/x86/dovecot-fts-xapian

### Related issue(s)
No

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [x] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

File: core/dovecot/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,4 @@
...
-WORKDIR /tmp
-RUN apk add git build-base automake autoconf libtool dovecot-dev xapian-core-dev icu-dev
-RUN git clone https://github.com/grosjo/fts-xapian.git \
-  && cd fts-xapian \
-  && git checkout 1.2.7 \
-  && autoreconf -vi \
-  && PANDOC=false ./configure --with-dovecot=/usr/lib/dovecot \
-  && make \
-  && make install
+ARG DISTRO=alpine:3.14
```
Comment (Diman0) [inline core/dovecot/Dockerfile:1]: Why did you change this?
We use pinned versions. Please change this back to 3.14.2.

**Review by Diman0 — CHANGES_REQUESTED** [review]
See inline comment. 
You must also add a [changelog entry](https://mailu.io/master/contributors/workflow.html#changelog). As the file name you can use the PR number.

File: core/dovecot/Dockerfile (hunk 1 above)
Comment (willofr) [reply core/dovecot/Dockerfile]: I think it's more interesting to follow the major version (3.14) than a minor version, so the project follows the evolutions of the underlying distribution (bug fixes, no major evolution). The latest 3.14 version of alpine is 3.14.3.
I rollback and propose to open a discussion on the subject ?

**Comment by willofr** [comment]
Revert back to alpine 3.14.2 as requested
Add changelog entry

**Comment by Diman0** [comment]
It looks good to me :+1: 
I will do some testing later this week and will then approve it.

**Review by nextgens — APPROVED** [review]
LGTM; I think it's working here ... but I'm not 100% sure it's a server search that's started.

**Review by Diman0 — APPROVED** [review]
